cli/objects/rule.py

```diff
@@ -198,31 +192,28 @@ def _get_jira_component(self, rule_dict: dict[Any, Any]) -> Optional[list[str]]:
             Optional[list[str]]: A list of strings representing the component(s) defined in a firewatch rule. If not defined, return None
         """
         components = []
-        if "jira_component" in rule_dict.keys():
-
-            jira_component = rule_dict["jira_component"]
-
-            if isinstance(jira_component, str):
-                components.append(jira_component)
-                return components
-            elif isinstance(jira_component, list):
-                for component in jira_component:
-                    if isinstance(component, str):
-                        components.append(component)
-                    else:
-                        self.logger.error(
-                            f'Component "{component}" in "jira_component" is not a string in firewatch rule: "{rule_dict}"',
-                        )
-                        exit(1)
-                return components
-            else:
-                self.logger.error(
-                    f'Value for "jira_component" must be either a list of strings (multiple components) or a string value (single component) in firewatch rule: "{rule_dict}"',
-                )
-                exit(1)
-
+        jira_component = rule_dict.get("jira_component")
+
+        if isinstance(jira_component, str):
+            components.append(jira_component)
+            return components
+        elif isinstance(jira_component, list):
+            for component in jira_component:
+                if isinstance(component, str):
+                    components.append(component)
+                else:
+                    self.logger.error(
+                        f'Component "{component}" in "jira_component" is not a string in firewatch rule: "{rule_dict}"',
+                    )
+                    exit(1)
+            return components
+        elif not jira_component:
+            return jira_component
         else:
```

Comment:
When this `else` called? 
you have `jira_component = rule_dict.get("jira_component")` which wll be either `str` or `list` or `None`
and you have
```
elif not jira_component:
            return jira_component
```
what do I miss?

Reply:
If someone uses `"jira_component": 3` or `"jira_component": True` for example. Sounds dumb, but I'd rather give them an error message.